### PR TITLE
Packaging: Add tendrl-api-httpd dependency

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-bug-id: Tendrl/api#118
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

File: tendrl-dashboard.spec [hunk 1]
```diff
@@ -9,6 +9,7 @@ Source0: %{name}-%{version}.tar.gz
 URL: https://github.com/Tendrl/dashboard
 
 Requires: tendrl-api
```
Comment (brainfunked) [inline tendrl-dashboard.spec]: Why is the tendrl-api package a dependency?

Comment (TimothyAsirJeyasing) [inline tendrl-dashboard.spec]: tendrl-frontend is not using any api from tendrl-api. However tendrl-frontend needs tendrl-api-httpd package which does some common configuration for both api and tendrl-frontend.

Comment (mbukatov) [inline tendrl-dashboard.spec]: Wouldn't make more sense to require `tendrl-api-httpd` instead then?

Comment (TimothyAsir) [inline tendrl-dashboard.spec]: yes, done the change as suggested

**Comment by gnehapk** [comment]
No longer needed. Hence, closing this issue.

**Pull request closed without merging**